packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+
+Time sensitive multipliers that can decrease fees and increase rewards to incentivize proposals and proofs when there are unpredictable deterrents acting against the engagement of proposers or provers.
+
+## Intrinsic Validity Function
+
+A function that is run on each proposed block to determine if the block contains valid data. If the block doesn’t contain valid data the block will be seen as a block with an empty transaction list.
+
+## Invalid Block
+
+A block that fails to pass the [Intrinsic Validity Function](#Intrinsic-Validity-Function).
+
+## Metadata (of the block)
+
+A tuple of 9 items comprising: block id, beneficiary (20-byte address), gas limit, timestamp, mixHash value, extraData value, Keccak-256 hash, the enclosing L1 block’s parent block number, the enclosing L1 block’s parent block hash.
+
+## Verified block (on-chain verified)
+
+A block is on-chain verified when its proof is submitted and its parent block is verified.
+
+## Parallel Proof
+
+Blocks can be proven in parallel because all intermediate states between blocks are known and [deterministic](#Deterministic).
+
+## Proposing a block
+
+Any willing entity can propose new Taiko blocks using the [TaikoL1 contract](#TaikoL1-Contract). Blocks are appended to a list in the order they come in (which is dictated by Ethereum L1 validators/block builders). Once the block is in the list it is [verified](#Verified-block-(on-chain verified)) and nodes can apply its state to the latest L2 state. A proposed block in Taiko is the collection of information, [Metadata](#Metadata), and a list of transactions.
+
+## RLP decodable
+
+A standard for data transfer between nodes in a space-efficient manner.
+
+## Signal Service
+
+A smart contract that can be used to exchange [cross-chain messages](<#Cross-chain-communication-(messaging)>) between L1 ↔ L2 and L2 ↔ L2 that any dapp developer can use.
+
+## Slot-availability multipliers
+
+Multipliers dependent on the number of unverified blocks aimed at offering the lowest fees and rewards when there are only a few unverified blocks and a surplus of available slots, and, on the opposite, as the number of available slots decrease, competition for the remaining slots increase for proposers, which leads to higher fees.
+
+## TaikoL1 Contract
+
+The smart contract on Ethereum L1 used to [propose](#Proposing-a-block), prove, and [verify](#Verified-block-(on-chain verified)) L2 blocks.
+
+## TaikoL2 Contract
+
+The smart contract on Taiko L2 that facilitates (i) [anchoring](#Anchor-Transaction) (ii) proving that a [proposed block](#Proposing-a-block) is [invalid](#Invalid_Block).
+
+## Timestamp (of the block)
+
+The timestamp used in the block, set to the enclosing L1 timestamp. If there are multiple L2 blocks included in one L1 block, their timestamp will be the same.
+
+## TokenVault
+
+A special contract in the Genesis Block with 2 to the power of 128 Ether that allows users to [bridge](#Bridge) Ether from and to Taiko.
```

Comment:
this actually just holds only ERC-20 tokens. the huge amount in EtherVault is 2**128 Ether pre-minted.

Reply:
i honestly feel like we can remove this definition but also fine with it staying. mainly because TokenVault is not part of the "core protocol" like "EtherVault" is. it's not *critical* to mention because it's a part of the bridge dapp which is built on top of the core protocol